### PR TITLE
Add quickstartWithHelmInKubernetes

### PR DESCRIPTION
**Pull request by Baiyelovecode**
Please do not create a Pull Request without creating an issue first. 

## What is the purpose of the change

Add quickstartWithHelmInKubernetes

## Verifying this change

Follow this checklist to help us incorporate your contribution quickly and easily:

- [x] Make sure there is a Github issue filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue. 
- [ ] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
- [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [ ] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/apache/rocketmq/tree/master/test).
- [ ] Run `mvn -B clean apache-rat:check findbugs:findbugs checkstyle:checkstyle` to make sure basic checks pass. Run `mvn clean install -DskipITs` to make sure unit-test pass. Run `mvn clean test-compile failsafe:integration-test`  to make sure integration-test pass.
- [ ] If this contribution is large, please file an [Apache Individual Contributor License Agreement](http://www.apache.org/licenses/#clas).

### CODE REVIEW

File: docs/01-quickstart/04quickstartWithHelmInKubernetes.md [hunk 1]
```diff
@@ -0,0 +1,180 @@
+### QuickStart
+本文介绍在kubernetesk环境下如何使用Helm快速部署一个单节点单副本 RocketMQ 服务，并完成简单的消息收发。
```
Comment (RongtongJin) [inline docs/01-quickstart/04quickstartWithHelmInKubernetes.md]: 单副本就行，可以去掉单节点

File: docs/01-quickstart/04quickstartWithHelmInKubernetes.md [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+
+#### 前提条件
+
+- 一个运行中的 `Kubernetes` 集群
+- 已安装的 `Helm`
```
Comment (RongtongJin) [inline docs/01-quickstart/04quickstartWithHelmInKubernetes.md]: 加一下helm版本的要求

**Comment by Baiyelovecode** [comment]
已修改，并增加了rocketmq-4.x

File: docs/01-quickstart/04quickstartWithHelmInKubernetes.md [hunk 3]
```diff
@@ -0,0 +1,193 @@
+### QuickStart
+本文介绍在kubernetesk环境下如何使用Helm快速部署一个单节点 RocketMQ-5.X 版本的服务，并完成简单的消息收发。
```
Comment (RongtongJin) [inline docs/01-quickstart/04quickstartWithHelmInKubernetes.md:2]: image
5.x的放在这个目录下

**Comment by Baiyelovecode** [comment]
Update quickstartWithKubernetes

**Pull request closed without merging**